### PR TITLE
Be more precise on how to set up and use the gem

### PR DESCRIPTION
**Pull request by kat-crumbs**

### CODE REVIEW

**Comment by psychocandy** [comment]
@kat-crumbs thanks!

File: README.md [hunk 1]
```diff
@@ -34,7 +46,27 @@ Naming is the same as in HasOffers documentation, also if it requires attributes
...
+HasOffersV3::Affiliate.update_payment_method_wire({ affiliate_id: '877', data: {} })
+```
+
+  or when working with forms:
+
+```ruby
+HasOffersV3::Advertiser.signup({
+  account: {
+    company: params[:company_name],
+    country: params[:company_country,
```
Comment (antlypls) [inline README.md]: Actually there is one small typo, missing closing bracket `]`.

Comment (psychocandy) [inline README.md]: Good catch!

Comment (kat-crumbs) [inline README.md]: upsi, sorry. Can you just fix it or should I submit a new PR?

Comment (miry) [inline README.md]: fixed

Comment (kat-crumbs) [inline README.md]: great, thx

**Pull request merged**